### ISSUE TITLE
Delete All Notification option in Notification center🚀 Feature: 

### ISSUE THREAD

**Issue by Vivek-PawarCG** [novuhq/novu#3240]
### 🔖 Feature description

In the notification center there are sometimes too many notifications and deleting them one by one is tiresome, so there should be an option to delete all notifications which will be a big relief.

### 🎤 Why is this feature needed ?

Deleting notifications one by one is time-consuming.

### ✌️ How do you aim to achieve this?

there should be button as 'delete all' similar to 'mark all as read'  in the notification center.

### 👀 Have you spent some time to check if this feature request has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Contributing Guidelines](https://github.com/novuhq/novu/blob/main/CONTRIBUTING.md)

### Are you willing to submit PR?

None

**Comment by gitstart** [comment]
@Vivek-PawarCG I would like to pick this up cc @scopsy

**Comment by scopsy** [comment]
@gitstart unfortuently this is not ready from the design standpoint. Would need to sync with @nknyshov about this one before. 🙏

**Comment by david-morris** [comment]
Hey I want to piggyback onto this ticket. I'm using the react hook API and have a lot of notifications that get deleted rapidly/in a batch. 
I need access to the full feed in my use case, so all pages get refetched.

I'm fine with debouncing that myself, but I need to have a way to batch deletes.  As it stands, the roundtrips take far too long and the user will generally navigate away from the page before all the deleting is finished.

Because of some other bugs, that means I have to unmount my provider, and the action won't finish.  The end effect on the UI is very frustrating, and I'm concerned that if we get customer complaints, we might have to migrate away from using Novu for this use case.

**Comment by ainouzgali** [comment]
Hi @Vivek-PawarCG @david-morris , an api call that deletes all messages/deletes all messages for a feed  should be merged very soon in [this PR ](https://github.com/novuhq/novu/pull/3523). 
@david-morris does that look like it could help your usecase?
cc @jainpawan21

**Comment by david-morris** [comment]
>  @david-morris does that look like it could help your usecase?

No, unfortunately. In my use case, I want to delete messages in a field that the user has *definitely seen*. As the user scrolls through the view, they will see different messages, and when one lingers for at least a second, it's considered definitely seen. So I want to be able to submit a list of IDs that I debounce.

Since the messages are organized based on searches on data that the message keys onto, it's not possible to lean on pagination or any single query in the message in my use case.

**Comment by rhinonix** [comment]
@ainouzgali Has there been any movement on this?

**Comment by jainpawan21** [comment]
@rhinonix 

We added [useRemoveAllNotifications](https://docs.novu.co/notification-center/client/react/api-reference#useremoveallnotifications) hook in react notification center package
